### PR TITLE
Azure AD Cloud Connect note

### PR DESCRIPTION
**Pull request by mprentice22153**
I believe this is a true statement:
Azure Active Directory Cloud Connect does not support hybrid migrations due to the inability to handle Exchange hybrid writeback.
If so, it would be good to have it clearly stated.
https://docs.microsoft.com/en-us/azure/active-directory/cloud-sync/what-is-cloud-sync#comparison-between-azure-ad-connect-and-cloud-sync

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/exchange-hybrid.md [hunk 1]
```diff
@@ -95,6 +95,9 @@ A hybrid deployment involves several different services and components:
 
    Learn more at: [Prerequisites for Azure AD Connect](/azure/active-directory/hybrid/how-to-connect-install-prerequisites).
 
+> [!NOTE]
+   > Azure Active Directory Cloud Connect does not support hybrid migrations due to the inability to handle Exchange hybrid writeback.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/exchange-hybrid.md:99]:
```suggestion
   > Azure AD Connect cloud sync does not support hybrid migrations due to its inability to handle Exchange hybrid writeback.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**